### ISSUE TITLE
Medium Priority WCAG Level AA

### ISSUE THREAD

**Issue by markrowen** [digital-land/digital-land#720]
Medium priority tickets with WCAG Level AA.

For the service to fall in line with WCAG 2.2 requirements, all A and AA issues must be resolved.

The following section contains areas that failed to meet WCAG 2.2 AA. For the service to fall 
in line with WCAG 2.2 requirements, all A and AA issues must be resolved. 

DAC Accessibility Audit Report is here: https://mhclg.sharepoint.com/:b:/s/DigitalPlanning/IQDZqGxgf4FCR5OlJZFA-tC_Abf_FUYF6Ydsr2FjVW7t4DQ?e=FaOLFI
